project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_forms.scss

```diff
@@ -92,3 +92,23 @@
 .panel-title {
   text-transform: uppercase;
 }
+
+#views-form-c4m-overview-og-members-page {
+
+  .vbo-fieldset-select-all {
+    width: 250px;
+    text-align: left;
+    padding-top: 20px !important;
```

Comment:
@ordavidil 
Originally, it was supposed to override another class with !important there.
But I rechecked, and it's not really required.
I'll remove it.